src/main/java/com/smashingmods/reroll/util/MalekUtil.java

```diff
@@ -0,0 +1,84 @@
+package com.smashingmods.reroll.util;
+
+import com.google.common.collect.AbstractIterator;
+import com.smashingmods.reroll.config.Config;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockLeaves;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public class MalekUtil {
```

Comment:
Change class name to something relevant to what it's doing. For example: PositionValidator

Reply:
Feel free to add a multiline author comment to claim your work.